### PR TITLE
Accept `client_id` as an alias of `visit_id`

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla-services/stubattribution/issues/153

---

The visit ID field is in fact the Google Analytics 'client' ID.

Note that we simply rename the field internally, we do not change the log statements to avoid breaking things (used by the DS team).

### CODE REVIEW

File: stubservice/stubhandlers/redirect.go [hunk 1]
```diff
@@ -103,11 +103,11 @@ func (s *redirectHandler) ServeStub(w http.ResponseWriter, req *http.Request, co
 	}
 
 	stubLocation := s.CDNPrefix + key
-	stubLocationUrl, err := url.Parse(stubLocation)
+	stubLocationURL, err := url.Parse(stubLocation)
 	if err != nil {
 		return errors.Wrap(err, "url.Parse")
 	}
-	http.Redirect(w, req, stubLocationUrl.String(), http.StatusFound)
+	http.Redirect(w, req, stubLocationURL.String(), http.StatusFound)
```
Comment (willdurand) [inline stubservice/stubhandlers/redirect.go:110]: unrelated changes because my local linter was complaining..

**Review by cvalaas — APPROVED** [review]
